blocks/nautilus/ff_container/ff_container-dashboard-section/ff_container-dashboard-section.less

```diff
@@ -1,6 +1,3 @@
-.ff_container-dashboard-section+.ff_container-dashboard-section {
-    margin-top: 22rem/@ff_base;
```

Comment:
The vertical margin actually exists as part of the ff_grid-columnar, where all children of the column container receive an equal vertical spacing.